api/v1beta1/network_disruption_test.go

```diff
@@ -187,6 +187,7 @@ var _ = Describe("NetworkDisruptionSpec", func() {
 						HTTP: &NetworkHTTPFilters{
 							Paths: paths,
 						},
+						Drop: 100,
```

Comment:
Why adding this?

Reply:
The previous spec was not actually valid, because it didn't define any packet disruption type. The disruptionSpec.Validate() would succeed, because we weren't checking the ddmark rules there. But now we are using spec.Validate to confirm the rules that used to only be checked by ddmark, so this spec (and a few others across the unit tests) need to have missing required fields added in. I believe I've done so in a way that respected those original unit tests' intent